wsd/COOLWSD.cpp

```diff
@@ -2560,9 +2552,40 @@ void COOLWSD::innerInitialize(Application& self)
 #endif // CODE_COVERAGE
 
     // Setup the jails.
+    const bool UseMountNamespaces = getConfigValue<bool>(conf, "mount_namespaces", false);
```

Comment:
Bit of a surprise here; do we really need to do this inside coolwsd ? I'd expect this to be specific to the forkit; or are we containing the whole coolwsd now as well ? =) [ which makes good sense if I think about it too ;-]

Reply:
In coolwsd we use setupChildRoot and that has a testing bind mount+umount of cool_test_mount to see if the full combination works and if it fails then disables bindmounting globally. So stuck that into the same type of combination of namespaces as used in the forkit.

Reply:
Makes sense I guess =) So we're not changing coolwsd's view of the world / capabilities etc.

Quite possibly coolwsd should (after sucking in its file data) enter a namespace with no system file-system visibility either =) but that's for later - I wonder what best practice is there these days.